services/app-api/resources/ssm-params.yml

```diff
@@ -21,44 +21,44 @@ Resources:
       Type: StringList
       Description: Branch specific email address of recipient for cms spa form submission emails
       Name: !Sub /configuration/${self:custom.stage}/email/cms_spa_form_cms_email
-      Value: "OneMAC@cms.hhs.gov"
+      Value: "\"OSG\" <OneMAC@cms.hhs.gov>"
 
   CmsSpaFormChipEmail:
     Type: AWS::SSM::Parameter
     Properties:
       Type: StringList
       Description: Branch specific email address of recipient for cms chip spa form submission emails
       Name: !Sub /configuration/${self:custom.stage}/email/cms_spa_form_chip_email
-      Value: "OneMAC@cms.hhs.gov"
+      Value: "\"CHIP to\" <OneMAC@cms.hhs.gov>"
 
   CmsSpaFormChipCcEmail:
     Type: AWS::SSM::Parameter
     Properties:
       Type: String
       Description: Branch specific email addresses of recipients for CC on cms chip spa form submission emails
       Name: !Sub /configuration/${self:custom.stage}/email/cms_chip_form_cc_email
-      Value: "OneMAC@cms.hhs.gov"
+      Value: "\"CHIP cc List\" <OneMAC@cms.hhs.gov>"
 
   CmsDpoEmail:
     Type: AWS::SSM::Parameter
     Properties:
       Type: String
       Description: Branch specific email addresses of the DPO group for CMS
       Name: !Sub /configuration/${self:custom.stage}/email/cms_dpo_email
-      Value: "OneMAC@cms.hhs.gov"
+      Value: "\"DPO group\" <OneMAC@cms.hhs.gov>"
 
   CmsDmcoEmail:
     Type: AWS::SSM::Parameter
     Properties:
       Type: String
       Description: Branch specific email addresses of the DMCO group for CMS
       Name: !Sub /configuration/${self:custom.stage}/email/cms_dmco_email
-      Value: "OneMAC@cms.hhs.gov"
+      Value: "\"DMCO group\" <OneMAC@cms.hhs.gov>"
 
   CmsDhcbsooEmail:
     Type: AWS::SSM::Parameter
     Properties:
       Type: String
       Description: Branch specific email addresses of the DHCBSOO group for CMS
       Name: !Sub /configuration/${self:custom.stage}/email/cms_dhcbsoo_email
-      Value: "OneMAC@cms.hhs.gov"
\ No newline at end of file
+      Value: "\"DHCBSOO group\" <OneMAC@cms.hhs.gov>"
```

Comment:
I think this exposes a flaw we have known about with the configuration of these emails in SSM and using defaults in lower environments such that by changing the value here it will rebuild the SSM parameter and we will have to manually set the proper values again when it gets to PROD.

I do realize the change was likely so you could tell which default email was actually being assigned since there are now multiple groups being CC'd, so I do like the change.

Reply:
Oh.... great point and that totally sucks :( - and yes, I did it so we could tell which actual email variable was in play because they are all the same default ...